.env

```diff
@@ -0,0 +1 @@
+API_AUTH_SECRET=tYTHmJmbSUJsqjHLyYyctK2YnK6vRaPUkWwXtE951
```

Comment:
I'd say we should keep the current behavior of committing changes to `.env`, as this follows the guidance of the official [Next.js docs](https://nextjs.org/docs/basic-features/environment-variables), which state that `.env.local` should be used to store secrets (it is being ignored). `.env`, `.env.development` and `.env.production` are meant for public environment variables only (URLs etc).

Reply:
Oh yeah you are right. My mistake.. I should have put the secret in a `.env.local` file instead of `.env`